src/equations/2d/compressible_euler.jl

```diff
@@ -32,6 +32,24 @@ function initial_conditions_density_pulse(x, t, equation::CompressibleEulerEquat
   return @SVector [rho, rho_v1, rho_v2, rho_e]
 end
 
+"""
+    initial_conditions_density_wave(x, t, equation::CompressibleEulerEquations2D)
+
+test case for stability of EC fluxes from paper: https://arxiv.org/pdf/2007.09026.pdf
+domain [-1, 1]^2
+mesh = 4x4, polydeg = 5
+"""
+function initial_conditions_density_wave(x, t, equation::CompressibleEulerEquations2D)
```

Comment:
I think you should add a test for this new IC to keep up coverage.

Reply:
This ic is used for the jacobian test...

Reply:
Sorry, my mistake!